source/documentation/using_the_api/getting_updates.md

```diff
@@ -6,106 +6,38 @@ You can regularly compare this value with new queries to a register to see if th
 
 ### Return all new entries since your last update
 
-You can use the [`GET /entries/` endpoint](#getentries) with the `start` and
-`limit` parameters to return specific entries to a register. 
+You can use the [`GET /entries/` endpoint](#getentries) to return specific entries to a register. This takes the parameters `start` and `limit`. 
 
-For example, your last highest entry number from the `country` register could
-be 205. You could then use the following request to find out if there are any
-new entries:
+For example, your last highest entry number from the `government-organisation` register could
+be 750. You could then make this request to find out if there are any new entries:
 
 ```http
-GET /entries/?start=206 HTTP/1.1
-Host: country.register.gov.uk
+GET /entries/?start=751 HTTP/1.1
+Host: goverment-organisation.register.gov.uk
 Accept: application/json
 Authorization: YOUR-API-KEY-HERE
 ```
 
-Example response:
+If you receive a response, you'll know there are updates.
```

Comment:
You will always receive a response, and a 200 one. If there are no new entries you'll get an empty collection:

```
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 2
Link: <?start=1990&limit=100>; rel="previous"

[]
```